app/models/miq_region.rb

```diff
@@ -62,7 +62,7 @@ def miq_servers
   end
 
   def servers_for_settings_reload
-    miq_servers.where(:status => "started")
+    MiqServer.in_my_region.where(:status => "started")
```

Comment:
Won't this unnecessarily queue a settings reload for the servers `in_my_region` if some other region is updated?